openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -297,6 +297,7 @@
     "edit-entity-name": "Edit {{entityType}}: \"{{entityName}}\"",
     "edit-glossary-display-name": "Edit Glossary Display Name",
     "edit-glossary-name": "Edit Glossary Name",
+    "edit-task": "Edit task",
```

Comment:
There is "edit-entity" key present, instead of adding this key, we can reuse that